beekeeper-api/pom.xml

```diff
@@ -20,6 +24,28 @@
       <groupId>org.springframework.boot</groupId>
       <artifactId>spring-boot-actuator</artifactId>
     </dependency>
+    <dependency>
+      <groupId>com.expediagroup</groupId>
+      <artifactId>beekeeper-core</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>net.kaczmarzyk</groupId>
+      <artifactId>specification-arg-resolver</artifactId>
+      <version>${specification-arg-resolver.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.json</groupId>
+      <artifactId>json</artifactId>
+      <version>20080701</version>
```

Comment:
Is this what resolved your JSON dependency issue?

Reply:
yas queen